docs/pages/en-us/Getting-Started.md

```diff
@@ -349,26 +347,26 @@ When passed as the WorkflowId on a task to the workflow worker, this workflow sp
 * An item should have its metadata extracted (including text for text-based files).
 * If it is an image, then the item will have OCR performed to extract its text.
 * Then the text will have its language detected.
-* The result describing the processed item should then be output to a queue so that the result may be examined.
+* The result describing the processed item should then be indexed into Elasticsearch so that the result may be examined.
 
 #### Message Format
 
-The format of the input task data is described in detail in the GitHub Repository for the policy worker library which is used in the workflow worker [here](https://github.com/CAFDataProcessing/worker-policy) under 'worker-policy-shared'. In the context of data processing the key properties to highlight are;
+The format of the input task data is described in detail in the GitHub Repository for the Document Worker Framework [here](https://github.com/CAFDataProcessing/worker-document/tree/develop/worker-document-shared#composite-document-handling) under 'Composite Document Handling'. The workflow worker specific properties are described [here](https://github.com/CAFDataProcessing/worker-workflow/tree/develop/worker-workflow-container#input-task-custom-data). In the context of data processing the key properties to highlight are;
```

Comment:
Can we link to the tag where possible so that the links are more stable - i.e. use `https://github.com/CAFDataProcessing/worker-document/tree/v3.3.0/worker-document-shared#composite-document-handling` rather than linking to the latest develop in case it changes.  Probably not possible for the other link since it hasn't been released.

Reply:
Updated worker-document link. I added a comment on the chained worker epic about going back and updating links to workflow worker in the documentation once the worker is released.